server_status/tests/urls.py

```diff
@@ -1,10 +1,14 @@
 """URLs to run the tests."""
-from django.conf.urls import include, url
+try:
```

Comment:
Sorry, one more thing. We can get rid of this `try` block if we are only supporting Django 1.10 and later. Do you think that's a reasonable assumption?

Reply:
I added that because of failures in the 1.10 and 1.11 versions of the tox run so I think we need to keep it in there until we're on >=2.x only.

Reply:
Oh right, `url` is supported in Django 1.10 but it looks like `include` wasn't added until 2.0